crates/cli/src/opts/global.rs

```diff
@@ -16,39 +16,47 @@ pub struct GlobalOpts {
     /// - 3 (-vvv): Print execution traces for failing tests.
     /// - 4 (-vvvv): Print execution traces for all tests, and setup traces for failing tests.
     /// - 5 (-vvvvv): Print execution and setup traces for all tests.
-    #[clap(short, long, global = true, verbatim_doc_comment, conflicts_with = "quiet", action = ArgAction::Count, help_heading = "Display options")]
-    pub verbosity: Verbosity,
+    #[arg(help_heading = "Display options", global = true, short, long, verbatim_doc_comment, conflicts_with = "quiet", action = ArgAction::Count)]
+    verbosity: Verbosity,
 
     /// Do not print log messages.
-    #[clap(short, long, global = true, alias = "silent", help_heading = "Display options")]
+    #[arg(help_heading = "Display options", global = true, short, long, alias = "silent")]
     quiet: bool,
 
     /// Format log messages as JSON.
-    #[clap(
-        long,
-        global = true,
-        alias = "format-json",
-        conflicts_with_all = &["quiet", "color"],
-        help_heading = "Display options"
-    )]
+    #[arg(help_heading = "Display options", global = true, long, alias = "format-json", conflicts_with_all = &["quiet", "color"])]
     json: bool,
 
     /// The color of the log messages.
-    #[clap(long, global = true, value_enum, help_heading = "Display options")]
+    #[arg(help_heading = "Display options", global = true, long, value_enum)]
     color: Option<ColorChoice>,
+
+    /// Number of threads to use. Zero specifies the number of logical cores.
+    #[arg(global = true, long, short = 'j', visible_alias = "jobs")]
+    threads: Option<usize>,
```

Comment:
```suggestion
    /// Number of threads to use. Specifying zero defaults to the number of logical cores.
    #[arg(global = true, long, short = 'j', visible_alias = "jobs")]
    threads: Option<usize>,
```

For context for to other reviewers: https://docs.rs/rayon/latest/rayon/struct.ThreadPoolBuilder.html#method.num_threads

> If num_threads is 0, or you do not call this function, then the Rayon runtime will select the number of threads automatically